### ISSUE TITLE
Standard color profiles for Tahoma render output 

### ISSUE THREAD

**Issue by Codeoloketuyi** [tahoma2d/tahoma2d#1436]
Good day, I have a feature request that I would love to see in Tahoma render output settings and that is a standard color profile like, sRGB, rec.709, and DCI-P3 .
Why I am asking is that I have noticed when I render out my project in Tahoma on my computer and import that project into my phone to view it . the colors of my animation project changed drastically like if In Tahoma the color are saturated on phone displays the color will be way to vibrant for my eyes to handle the same gose for if it has high contrast it will be way brighter.

 So what this color profile dose is to maintain the color space of the animation and to make it compatible for other devices.

 Thanks for reading ☺️

**Describe the solution you'd like**

**Comment by Codeoloketuyi** [comment]
Here is an example.
![Uploading CM.0001.png…]()
![Uploading untitled8.0001.png…]()

**Comment by artisteacher** [comment]
Yes! This is often a noticeable issue for macs, since they use P3-1600 or P3-500 by default. I think this should be possible to enable for at least the ffmpeg formats.

[https://trac.ffmpeg.org/wiki/colorspace](url)

**Comment by Codeoloketuyi** [comment]
> Yes! This is often a noticeable issue for macs, since they use P3-1600 or P3-500 by default. I think this should be possible to enable for at least the ffmpeg formats.
> 
> [https://trac.ffmpeg.org/wiki/colorspace](url)

Thank you I was frustrated when I found out my project animation colors were changing
